### PR TITLE
Updating Javascript and fixing minor bugs

### PR DESCRIPTION
**Pull request by TafDev**
Pulled my last request because it had a bit of redundant code and also the counter was counting letters not words.

I couldn't understand this instruction: 

> Add some Javascript to the provided <script> tag in the views/index.erb file, that counts the number of words in each section. You ought to have at least twenty words for each of the questions.

because I did not see any questions.

### CODE REVIEW

File: primes.rb [hunk 1]
```diff
@@ -1,6 +1,21 @@
...
   def self.sum_to(limit = 100)
     # TODO - add your prime number solution here...
-    "I'm working on it!"
-  end
-end
\ No newline at end of file
+    primes = (2..limit).select do |n|
+    	(2..Math.sqrt(n)).none? do |i|
+    		(n%i).zero?
+    	end
```
Comment (dmgarland) [inline primes.rb]: I like the elegance of this nice one

File: primes.rb [hunk 2]
```diff
@@ -1,6 +1,21 @@
...
+    	sum = primes.inject { |sum, i| sum + i }
+	end
+end
+
+Primes.sum_to(100)
+Primes.sum_to(1000)
+Primes.sum_to(10000)
+
+#puts Primes.sum_to(2000000)
+#commented out the 2 million soloution as it takes a while to run but it does work
```
Comment (dmgarland) [inline primes.rb]: That's the trade off we need to make for larger numbers. Your approach is creating temporary arrays with all of the intermediate values in, which _looks_ great in Ruby code but suffers from being a bit inefficient. You'd either need to uppack this to use less OO loops, or take a look at what `lazy` does in the Ruby 2 Enumberable library.

File: primes.rb (hunk 1 above)
Comment (TafDev) [inline primes.rb]: thank you :)

File: primes.rb (hunk 2 above)
Comment (TafDev) [inline primes.rb]: I went and did a little more research into it using stackoverflow, google and the documentation. I found i can use the prime method which ruby already has. When I tested this method... it worked much faster and also looks a lot more tidy. Here it is below:

```
require 'prime'

class Primes
  def self.sum_to(limit)
    Prime.take_while {|p| p < limit}.reduce(:+)
  end 
end

 puts Primes.sum_to(10000)
 puts Primes.sum_to(1000)
 puts Primes.sum_to(100)
 puts Primes.sum_to(2000000)
```

Please let me know if you like for me to push this new method into my profile.

**Pull request closed without merging**